packages/remix-google-cloud-functions/__tests__/server-test.ts

```diff
@@ -0,0 +1,271 @@
+import supertest from "supertest";
+import { createRequest } from "node-mocks-http";
+import {
+  createRequestHandler as createRemixRequestHandler,
+  Response as NodeResponse,
+} from "@remix-run/node";
+import { Readable } from "stream";
+import { http } from "@google-cloud/functions-framework";
+import { getTestServer } from "@google-cloud/functions-framework/build/src/testing.js";
```

Comment:
@penx how do you feel about using this import? 

It's not pretty but it works without blocking this on the upgrade to jest 28

Reply:
I think we can keep this like this for now & change in the Jest 28 PR

Reply:
CI seems to be green, so let's keep the import as-is then. 👍